addons/ondevice-controls/src/Panel.tsx

```diff
@@ -0,0 +1,25 @@
+// import React, { useRef, useEffect } from 'react';
+import React from 'react';
+import { View } from 'react-native';
+
+// const usePrevious = (children) => {
```

Comment:
Could these commented-out portions be removed? 🤔

Reply:
I'll remove this.